python/ray/tune/tune.py

```diff
@@ -127,6 +127,39 @@ def _report_progress(
         reporter.report(trials, done, sched_debug_str, executor_debug_str)
 
 
+def _setup_signal_catching() -> threading.Event:
+    original_handler = signal.getsignal(signal.SIGINT)
+    stop_event = threading.Event()
```

Comment:
more specific name? `tune_loop_interrupted_event` or something alike?

Reply:
If you're ok with it I'd like to keep this, as we set up general signal catching here and not just the event. Let me know if you disagree!
Meanwhile I'll kick of tune cloud release tests

Reply:
oh I mean the event name can be more specific? The method name `_setup_signal_catching` sounds fine to me!

Reply:
Ah I see! Yes, thanks, will updated!